dashboard/config/scripts/test_external_markdown.external

```diff
@@ -1,54 +1,55 @@
 name 'Test External Markdown'
-encrypted 'l6Vrcl8nmHBhsGMJ+8t4wC4kR22KpUwHEeoaicMQhe8siR5nvkl85eUSXniF
```

Comment:
Does this level need to be updated? Its only used in a unit test

Reply:
I believe it's also used in a UI test?